cli/cmd/lib_cli_config.go

```diff
@@ -838,12 +840,19 @@ func removeEnvFromCLIConfig(envName string) error {
 		updatedEnvs = append(updatedEnvs, env)
 	}
 
-	if deleted == false && envName != types.LocalProviderType.String() {
+	if !deleted {
+		if envName == types.LocalProviderType.String() {
```

Comment:
If nothing was deleted and env is local, it likely means that cli config got into an erroneous state somehow. We should allow `writeCLIConfig(cliConfig)` because it will add the expected local config before writing.

Reply:
Agreed. Fixed!